plumbing/api.go

```diff
@@ -187,6 +187,11 @@ func (api *API) NetworkFindProvidersAsync(ctx context.Context, key cid.Cid, coun
 	return api.network.FindProvidersAsync(ctx, key, count)
 }
 
+// NetworkGetClosestPeers issues a getClosestPeers query to the filecoin network.
```

Comment:
@rosalinekarr @anorth once we kill the node api we might want to make a sweep of plumbing and move anything that is not of general utility out to a direct dependency of the command that uses it, eg this and some of the other network calls.